torch/onnx/_internal/exporter.py

```diff
@@ -45,7 +49,10 @@
 # 'import onnx' inside of dynamo_export (by way of _assert_dependencies).
 if TYPE_CHECKING:
     import onnx
-
+    import onnxscript  # type: ignore[import]
+    from onnxscript.function_libs.torch_lib import (  # type: ignore[import]
+        registration as torchlib_registry,
+    )
 
 _DEFAULT_OPSET_VERSION: Final[int] = 18
```

Comment:
Following on my last comment. Is it possible to delete this if it is not used anywhere else?